.vscode/settings.json

```diff
@@ -1,17 +1,12 @@
 {
```

Comment:
Sorry to be a pain, but can we just leave these config files alone please? Changing them in the repo really doesn't accomplish anything.

Reply:
ah yes - that one is on the top-level ... I guess we can remove this one and keep the ones in the subfolders (while for esp-hal-common we could have the templates there)

Reply:
Thanks, I'll rebase this on main once your PR is merged.